presto-main/src/test/java/com/facebook/presto/operator/TestAnnotationEngineForScalars.java

```diff
@@ -67,8 +64,8 @@
     @Description("Simple scalar with single implementation based on class")
     public static class SingleImplementationScalarFunction
     {
-        @SqlType(DOUBLE)
-        public static double fun(@SqlType(DOUBLE) double v)
+        @SqlType(StandardTypes.DOUBLE)
```

Comment:
You are change from `String` to `Type` here. Will this fail? 

Same argument for `BIGINT`.

Reply:
nvm. It looks confusing when reviewing two commits together.